### PR TITLE
Remove Dockerfiles from parent directory

### PR DESCRIPTION
**Pull request by khrm**
Remove Dockerfile from parent directory and add dockerfiles from .make directory to Makefile. 
Dockerfile in parent directory is auto generated one which doesn't build correct image and fails to run.
Run make image and podman run quay.io/openshiftio/fabric8-services-fabric8-webhook to test

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -58,10 +58,10 @@ REGISTRY_IMAGE = ${PROJECT_NAME}
 
 ifeq ($(TARGET),rhel)
 	REGISTRY_URL_IMAGE := ${REGISTRY_URI}/openshiftio/rhel-${REGISTRY_NS}-${REGISTRY_IMAGE}
-	CONTAINERFILE := Dockerfile.rhel
+	CONTAINERFILE := $(SELF_DIR)/.make/Dockerfile.rhel
 else
 	REGISTRY_URL_IMAGE := ${REGISTRY_URI}/openshiftio/${REGISTRY_NS}-${REGISTRY_IMAGE}
-	CONTAINERFILE := Dockerfile
+	CONTAINERFILE := $(SELF_DIR)/.make/Dockerfile
```
Comment (sthaha) [inline Makefile:64]: Does this mean you have to remove the `Dockerfile.rhel` as well just like you removed `Dockerfile` ?

**Review by sthaha — CHANGES_REQUESTED** [review]
Could you please add more details as to why this patch is needed in  commit message?

File: Makefile (hunk 1 above)
Comment (khrm) [reply Makefile:64]: No. Dockerfile in parent directory was auto-generated one not the one which I used for building image. Dockerfile.rhel is used for building rhel image.

Comment (khrm) [reply Makefile:64]: Removed Dockerfile.rhel from parent directory.

**Review by sthaha — APPROVED** [review]

**Pull request merged**